src/main/java/kostka/moviecatalog/service/ValidationService.java

```diff
@@ -0,0 +1,16 @@
+package kostka.moviecatalog.service;
+
+import kostka.moviecatalog.exception.InvalidDtoException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ValidationService {
```

Comment:
Is this supposed to be a validation service for all configs? I don't think this is  a good idea.
More cohesive solution would be to put the validation logic to each individual configs.